website/projects/tests/tests_aws/test_awssync_refactored.py

```diff
@@ -10,14 +11,134 @@
 
 from courses.models import Semester
 
+from mailing_lists.models import MailingList
+
 from projects.aws.awssync_refactored import AWSSyncRefactored
 from projects.aws.awssync_structs import AWSTree, Iteration, SyncData
+from projects.models import Project
 
 
 @mock_organizations
 class AWSSyncRefactoredTest(TestCase):
     def setUp(self):
+        """Set up testing environment."""
+        self.semester = Semester.objects.create(year=2023, season=Semester.SPRING)
+        for i in range(3):
+            self.mailing_list = MailingList.objects.create(address="test" + str(i))
+            self.project = Project.objects.create(
+                id=i, name="test" + str(i), semester=self.semester, slug="test" + str(i)
+            )
+            self.mailing_list.projects.add(self.project)
```

Comment:
Where are these used in your unit tests? Seems nowhere. I would also define such dummy values inside your unit tests, as that is their scope. Right now, these variables are created for each unit test.